queries.sql

```diff
@@ -1,10 +1,59 @@
-/*Queries that provide answers to the questions from all projects.*/
-
-SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT * FROM animals WHERE date_of_birth BETWEEN '1-1-2016' AND '31-12-2019';
-SELECT * FROM animals WHERE neutered = '1' AND escape_attempts < 3;
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
-SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = '1';
-SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+BEGIN;
+-- Start a Transaction:
+BEGIN
+
+-- Update "species" to "unspecified" for Animals with Names Ending in "mon":
+UPDATE
+  animals
+SET
+  species = 'unspecified'
+WHERE
+  name LIKE '%mon';
+
+UPDATE
+  6
+
+  -- Update "species" to "pokemon" for Animals without a Species:
+UPDATE
+  animals
+SET
+  species = 'pokemon'
+WHERE
+  species IS NULL;
+
+UPDATE
+  5 -- TAKE A BREATH 
+  BEGIN;
+
+BEGIN
+
+-- Delete Animals Born After Jan 1st, 2022:
+DELETE FROM
+  animals
+WHERE
+  date_of_birth > '2022-01-01';
+
+DELETE 1 SAVEPOINT before_weight_update;
+
+-- Create a Savepoint for the Transaction:
+SAVEPOINT
+UPDATE
+  animals
+SET
+  weight_kg = - weight_kg;
+
+UPDATE
+  10 ROLLBACK TO SAVEPOINT before_weight_update;
+
+ROLLBACK
+
+-- Update Animals' Weights That Are Negative:
+UPDATE
+  animals
+SET
+  weight_kg = - weight_kg
+WHERE
+  weight_kg < 0;
+
+UPDATE
+  4
```

Comment:
- [ ] As part of the requirement, you also missed the last part of the query which i screenshotted below. Kindly add it up 👍 

### Requirement:
![image](https://github.com/Mike47ip/vet-clinic-database/assets/50990925/a24bf699-0640-4220-b3f8-edea2e923431)